src/borg/archive.py

```diff
@@ -1424,19 +1450,22 @@ def process_file(self, *, path, parent_fd, name, st, cache, flags=flags_normal):
                     else:
                         status = "M" if known else "A"  # regular file, modified or added
                     self.print_file_status(status, path)
+                    self.stats.files_stats[status] += 1
```

Comment:
Just realized we have some more status keys:
- misc. for non-regular files, like d for dirs, s for symlinks, b/c for block/char devices, ...
- E for files borg could not read

I guess we should extend stats to some of them and ignore the less interesting ones:
- E we should definitely track
- maybe check the borg code and the docs if you find other interesting ones.

Reply:
I'm working on this right now, but I don't know how to purposefully damage a file

Reply:
Yeah, testing this is a bit tricky. But we already have tests that **mock** erroneous behaviour. See e.g. the xattr related tests.

Guess we could also just initialize the "E" counter to 0 and (considering it is the same code that also increments the "M" counter) just assume that it will also work.

Reply:
I took a look at the docs and I think maybe 'x' is important. Should I add it?

Reply:
maybe let's first focus on getting the other stuff working and the tests succeed.

`x` can be a bit difficult, depending on whether `-` or `!` is used to exclude, the counts there would be very different.